lib/defra_ruby_companies_house.rb

```diff
@@ -7,7 +7,7 @@ def initialize(company_no)
     @company_url = "#{Rails.configuration.companies_house_host}#{company_no.to_s.rjust(8, '0')}"
     @api_key = Rails.configuration.companies_house_api_key
 
-    load_company
+    StandardError unless load_company
```

Comment:
`raise StandardError unless load_company`

Reply:
![image](https://user-images.githubusercontent.com/91424856/173620672-fdc7af08-3f2e-4b61-a09e-145c25fc3899.png)
Putting raise in makes a lot of the rspec tests fail